src/ui/main_window.ui

```diff
@@ -303,14 +304,14 @@
    </property>
   </action>
   <action name="dactionInputLine">
-   <property name="text">
-    <string>Compact input line</string>
-   </property>
    <property name="checkable">
-     <bool>true</bool>
+    <bool>true</bool>
    </property>
    <property name="checked">
-     <bool>false</bool>
+    <bool>false</bool>
+   </property>
+   <property name="text">
```

Comment:
This change is just Git noise - if you can manually put it back to the original position it will disappear and make the changeset (*slightly*) smaller!

🤓